### PR TITLE
Add support for sber-zvuk.com

### PR DESCRIPTION
**Pull request by UnQuaiz**
**Describe the changes you made**
This request is going to add support for SberZvuk (sber-zvuk.com), audio streaming service, by setting up a new connector.

**Additional context**
RE: #2700

### CODE REVIEW

File: src/connectors/sber-zvuk-dom-inject.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const getTrackInfo = (track, currentTime) => {
```
Comment (alexesprit) [inline src/connectors/sber-zvuk-dom-inject.js:3]: Could you use function declarations?

File: src/connectors/sber-zvuk-dom-inject.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+		currentTime,
+		album,
+	};
+};
+
+const queueEvent = (event) => {
+	let type = null;
+	let trackInfo; let queue;
+
+	if (event.type === 'update') {
```
Comment (alexesprit) [inline src/connectors/sber-zvuk-dom-inject.js:22]: You can check for `event.type` first, and return earlier, if it's not equal "update". This allows to remove check if `type` (not event type) is not nullable.

File: src/connectors/sber-zvuk-dom-inject.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+			case 'currentTime':
+			case 'current_idx':
+				queue = event.name === 'current_idx' ? event.object : event.object.Queue;
+				trackInfo = getTrackInfo(queue.currentTrack, queue.Player.currentTime);
+				type = 'currentTime';
+				break;
+			case 'buffered':
+			case 'isLoading':
+			case 'initialPlaying':
+			case 'manually':
```
Comment (alexesprit) [inline src/connectors/sber-zvuk-dom-inject.js:38]: Are these cases required? I think we can omit them.

File: src/connectors/sber-zvuk-dom-inject.js (hunk 1 above)
Comment (UnQuaiz) [reply src/connectors/sber-zvuk-dom-inject.js:3]: sure

File: src/connectors/sber-zvuk-dom-inject.js (hunk 2 above)
Comment (UnQuaiz) [reply src/connectors/sber-zvuk-dom-inject.js:22]: event.type is not nullable. `type` filled by mobx observer, and always has some type of action. We interested in `update` action, when some property have changed. I'll fix checking.

File: src/connectors/sber-zvuk-dom-inject.js (hunk 3 above)
Comment (UnQuaiz) [reply src/connectors/sber-zvuk-dom-inject.js:38]: Yep. Forgot to cleanup

File: src/connectors/sber-zvuk-dom-inject.js (hunk 2 above)
Comment (alexesprit) [reply src/connectors/sber-zvuk-dom-inject.js:22]: > event.type is not nullable

I didn't say it's nullable. :)

My idea was to remove check for `type` value (which is from `mobx` observer). The `type` value is null, if we received an event with `event.type` != "update", so I suggested to check for `event.type` first, and return if it's not equal "update".

Comment (UnQuaiz) [reply src/connectors/sber-zvuk-dom-inject.js]: already done :)

**Review by alexesprit — APPROVED** [review]

**Comment by alexesprit** [comment]
👍🏻

**Pull request merged**